CHANGELOG.md

```diff
@@ -1,5 +1,179 @@
 
+## v0.14.0 (2017-3-27)
+
+Authentication via user roles, client-side debugging and numerous bug fixes.
+
+### Enhancements
+
+#### :gear: Authentication via user roles ([#243](https://github.com/DevExpress/testcafe/issues/243))
+
+Many test scenarios involve the activity of more than one user. TestCafe addresses these scenarios by providing a convenient way
+to isolate authentication test actions and apply them easily whenever you need to switch the user account.
+
+A piece of logic that logs in a particular user is called a *role*. It is a good practice to create a role for each user account participating in your test.
+
+Create roles via the `Role` constructor. You can keep them in a separate helper file.
+
+*helper.js*
+
+```js
+import { Role } from 'testcafe';
+
+export var regularAccUser = Role('http://example.com/login', async t => {
+    await t
+        .typeText('#login', 'TestUser')
+        .typeText('#password', 'testpass')
+        .click('#sign-in');
+});
+
+export var facebookAccUser = Role('http://example.com/login', async t => {
+    await t
+        .click('#sign-in-with-facebook')
+        .typeText('#email', 'testuser@mycompany.com')
+        .typeText('#pass', 'testpass')
+        .click('#submit');
+});
+
+export var admin = Role('http://example.com/login', async t => {
+    await t
+        .typeText('#login', 'Admin')
+        .typeText('#password', 'adminpass')
+        .click('#sign-in');
+});
+```
+
+In test code, use the `t.useRole` method to switch between roles.
+
+*test.js*
+
+```js
+import { regularAccUser, admin } from './helper';
+import { Selector } from 'testcafe';
+
+const entry        = Selector('#entry');
+const removeButton = Selector('#remove-entry');
+
+fixture `My Fixture`
+    .page `http://example.com`;
+
+test('test that involves two users', async t => {
+    await t
+        .useRole(regularAccUser)
+        .expect(entry.exists).ok()
+        .expect(removeButton.visible).notOk()
+        .useRole(admin)
+        .expect(removeButton.visible).ok()
+        .click(removeButton)
+        .expect(entry.exists).notOk()
+});
+```
+
+To learn more, see [User Roles](https://devexpress.github.io/testcafe/documentation/test-api/authentication/user-roles.html).
+
+#### :gear: Client-side debugging ([#918](https://github.com/DevExpress/testcafe/issues/918))
+
+We have added a new `t.debug` method to debug test behavior on the client.
+
+When test execution reaches `t.debug`, it pauses so that you can open browser's developer tools
+and check the web page state, DOM elements location, their CSS styles.
+
+```js
+fixture `My fixture`
+    .page `https://devexpress.github.io/testcafe/example`;
+
+test('My test', async t => {
+    await t
+        .debug()
+        .click('#populate')
+        .click('#submit-button');
+});
+```
+
+In the footer, you'll find buttons that allow you to continue test execution or step to the next test action.
+
+![Page Footer in the Debug Mode](media/client-debugging-footer.png)
```

Comment:
I guess we need for screenshot with new buttons to be landed? Or it can be updated independently?